package.json

```diff
@@ -34,14 +35,14 @@
     "rollup-pluginutils": "^1.5.0"
   },
   "peerDependencies": {
-    "babel-core": "6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc"
+    "@babel/core": "7 || ^7.0.0-beta"
   },
   "devDependencies": {
-    "babel-core": "7.0.0-alpha.17",
-    "babel-plugin-external-helpers": "7.0.0-alpha.17",
-    "babel-plugin-transform-decorators": "7.0.0-alpha.17",
-    "babel-plugin-transform-runtime": "7.0.0-alpha.17",
-    "babel-preset-es2015": "7.0.0-alpha.17",
+    "@babel/core": "^7.0.0-beta.5",
+    "@babel/plugin-external-helpers": "^7.0.0-beta.5",
+    "babel-plugin-transform-decorators": "^7.0.0-beta.3",
```

Comment:
This should be available on beta.5 too, just as `babel-plugin-proposal-decorators`

Reply:
It seems it isnt on npm https://www.npmjs.com/package/@babel/plugin-transform-decorators

> npm ERR! 404 Not Found: @babel/plugin-transform-decorators@7.0.0-beta.5

> npm ERR! notarget No matching version found for babel-plugin-transform-decorators@7.0.0-beta.5

Reply:
Yep! It is `proposal-` now, not `transform-`.

Reply:
Ah i missed that it was renamed.

Reply:
damn, forgot about that thing too 😄 gonna update PR in a minute